src/test/java/org/assertj/core/error/ShouldNotContainString_create_Test.java

```diff
@@ -81,4 +83,37 @@ public void should_create_error_message_with_several_string_values() {
                                    "but found:%n" +
                                    " <[\"ya\"]>%n"));
   }
+
+  @Test
+  public void should_create_error_message_for_ignoring_case() {
+    // GIVEN
+    ErrorMessageFactory factory = ShouldNotContainCharSequence.shouldNotContainIgnoringCase("Yoda", "OD");
+    // WHEN
+    String message = factory.create(new TextDescription("Test"), new StandardRepresentation());
+    // THEN
+    then(message).isEqualTo(format("[Test] %n" +
+                                   "Expecting:%n" +
+                                   " <\"Yoda\">%n" +
+                                   "not to contain (ignoring case):%n" +
+                                   " <\"OD\">%n"));
+  }
+
+  @Test
+  public void should_create_error_message_for_ignoring_case_with_multiple_findings() {
+    // GIVEN
+    ErrorMessageFactory factory = ShouldNotContainCharSequence.shouldNotContainIgnoringCase("Yoda",
+                                                                                            Arrays.array("OD", "da", "Luke"),
+                                                                                            Sets.newSet("OD", "da"));
```

Comment:
static import shouldNotContainIgnoringCase, array and newSet and see if everything can fit on one line